CommentingPractice/comment-practice.js

```diff
@@ -18,6 +19,8 @@ import * as martin from './services/spaces.js';
     //used my own function because it doesnt seem like anybody else added something
     var space = "space is large";
      console.log(amonte.randomlyCapitalize(space));
+    var word = "WORD";
+    console.log(amonte.lowercase(word));
```

Comment:
Your import statement at line 9 puts the lowercase function within the ramintha alias. If you were to run this version you'll see this error:

comment-practice.js → main.js...
[!] Error: Could not resolve './services/lowercase.js' from comment-practice.js